tools/targets/NCS.py

```diff
@@ -29,17 +32,17 @@ def ranges(i):
 
 
 def add_fib_at_start(arginput):
-    input_file = arginput + ".bin"
-    file_name_hex = arginput + "_fib.hex"
+    input_file = arginput + ".hex"
```

Comment:
You may want to avoid this assumption, but that can come with another PR.

If you change the code that calls this function (in tools/targets/__init__.py) to:
```python
class NCS36510TargetCode:
    @staticmethod
    def ncs36510_addfib(t_self, resources, elf, binf):
        from tools.targets.NCS import add_fib_at_start
        add_fib_at_start(binf)
```
You could correspondingly change this function prototype (and first few lines) to:
```python
def add_fib_at_start(input_file):
    basename, ext = os.path.splitext(input_file)
    assert(ext is ".hex", "Error: Post-bulid script input must be an intel-Hex format file")
    file_name_hex = basename + ".hex"
    file_name_hex = basename + ".bin"
```

This would be a bit safer, and a bit easier to understand when it fails.